common.blocks/button/_type/button_type_link.js

```diff
@@ -23,12 +23,16 @@ provide(Button.decl({ modName : 'type', modVal : 'link' }, /** @lends button.pro
         'disabled' : {
             'true' : function() {
                 this.__base.apply(this, arguments);
+
                 this.domElem.removeAttr('href');
+                this.domElem.attr('aria-disabled', true);
```

Comment:
1) Why do we do this only for `button_type_link`? What about `button` itself?
2) Can we do it in the `control` base class? Does this behaviour differ from block to block?